src/mn_setup.c

```diff
@@ -2653,7 +2653,10 @@ static setup_menu_t music_settings1[] = {
 
 static void UpdateGainItems(void)
 {
+#if defined (HAVE_FLUIDSYNTH)
     DisableItem(auto_gain, music_settings1, "fl_gain");
+#endif
+
```

Comment:
I don't think we need this extra blank line.

Reply:
I personally feel like the function's code would be too cramped without it, but I can remove it if you insist.

Reply:
> I can remove it if you insist.

I don't.